test/semantics/omp-clause-validity01.f90

```diff
@@ -377,6 +377,15 @@
     a = 3.14
   enddo
 
+! 2.13.1 master
+
+  !$omp parallel
+  !ERROR: NUM_THREADS clause is not allowed on the MASTER directive
```

Comment:
No clause is allowed on master construct, currently just a single case is written considering num_threads as clause , do we need all tests with all the remaining clauses?

Reply:
I think just checking one or a few is probably fine, writing one for every clause would be a large amount of effort for not a lot of gain